README.md

```diff
@@ -7,21 +7,32 @@
 * [Go - 1.15](https://golang.org/doc/go1.15)
 * [Docker](https://docs.docker.com/get-docker/)
 * [Docker Compose](https://docs.docker.com/compose/install/)
+* [NodeJS](https://nodejs.org/en/download/package-manager/)
 
+## Create necessary dot env files
+- Create `.env` file
+- Create env dependent files (depend on your env)
+  - `.env.release`
+  - `.env.debug` (for development)
+  - `.env.test`
+- Add values from `.env.example` (for env dependent variables eg. `DB_NAME` must add to dependent env  files)
```

Comment:
Let's add some space between each section for maintainability and follow the heading hierarchy (h2 -> h3 not h4)

```suggestion
- Add values from `.env.example` (for env dependent variables eg. `DB_NAME` must add to dependent env  files)

## Usage

### Run the Go application for development
```

Reply:
Fixed in 2f18e660051fb9921c8edd14b1a4c70a615a8f6e